### ISSUE TITLE
Add chemoreceptor organelle

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#2039]
https://forum.revolutionarygamesstudio.com/t/chemoreseptor-organelle/765

- [x] organelle added, and is placeable
- [x] when player cell has it, the tutorial line code is extended to create additional lines when needed
- [x] colour and compound type the chemoreseptor detects can be changed
- [ ] It'd be nice to have a disable condition, like "if I'm over 90% on glucose don't show the line to glucose", this will be a bit more advanced to do than the other things in this issue, so this can be left for later.

requires: https://github.com/Revolutionary-Games/Thrive/issues/2038

requires: https://github.com/Revolutionary-Games/Thrive/issues/2036
requires: https://github.com/Revolutionary-Games/Thrive/issues/2037
- [ ] as a follow up it'd be nice to make the AI get some benefit as well (check this as done once issue is opened)
